src/jvm/main/org/jetbrains/kotlinx/lincheck/util/UnsafeHolder.kt

```diff
@@ -34,4 +34,21 @@ internal inline fun <T> readFieldViaUnsafe(obj: Any?, field: Field, getter: Unsa
         val offset = UnsafeHolder.UNSAFE.objectFieldOffset(field)
         return UnsafeHolder.UNSAFE.getter(obj, offset)
     }
+}
+
+internal fun readField(obj: Any?, field: Field): Any? {
```

Comment:
Let's rename to `readFieldViaUnsafe` too (there should be no ambiguity, because the other function with the same name takes additional lambda argument).